packages/stream_chat_persistence/README.md

```
@@ -66,12 +73,10 @@ Due to Drift web (for offline storage) you need to include the sql.js library:
 </html>
 ```
 
-You can grab the latest version of sql-wasm.js and sql-wasm.wasm [here](https://github.com/sql-js/sql.js/releases) and copy them into your `/web` folder.
+You can grab the latest version of `sql-wasm.js` and `sql-wasm.wasm` [here](https://github.com/sql-js/sql.js/releases) and copy them into your `/web` folder.
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)